src/routes/acquirerRoutes.ts

```diff
@@ -91,6 +92,11 @@ router.get("/:resourceID/:step", async (req: Request, res: Response) => {
   const assetTitle = formdata.assetTitle;
   const contactPoint = formdata.contactPoint;
 
+  if (stepData.errorMessage) {
+    // I wanna see this log
+    console.error(`Step Error: ${stepData.errorMessage}`);
```

Comment:
do we need to keep this? Ok if purposely left in for debugging etc

Reply:
was left in for debugging, though I think I've hit all error cases in the validator. So this can be removed, thanks for flagging